### PR TITLE
fast-pt v3.2

### PR DESCRIPTION
**Pull request by dderienzo**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dderienzo** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml:48]: @jablazek must confirm he wants to be a maintainer here.

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
```

We don't list deps of deps.

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml]: Please use the python_min syntax in the hints.

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml:34]: Why did you remove this?

**Review by beckermr — CHANGES_REQUESTED** [review]
Thanks for the PR! It needs some fixes.

File: recipe/meta.yaml
Comment (dderienzo) [reply recipe/meta.yaml:34]: Apologies that one was an accident.

File: recipe/meta.yaml
Comment (dderienzo) [reply recipe/meta.yaml]: I added, let me know if I did it right or not.

**Comment by dderienzo** [comment]
I added suggested edits. Thank you for working through this with me, I am super unfamiliar with this as you can probably tell.

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
    - python {{ python_min }}
```

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml:34]:
```suggestion
    - pip
    - python {{ python_min }}
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml
Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
```

I doubt you actually want conda to be able to install this package with python 3.5 in conda.

**Comment by beckermr** [comment]
once @jablazek confirms, we can merge

File: recipe/meta.yaml
Comment (jablazek) [reply recipe/meta.yaml:48]: I confirm - I would like to be a maintainer. (Apologies for the delay. I thought I had confirmed already, but I think it was on a different repo). Or did you mean confirm the overall PR?

Comment (beckermr) [reply recipe/meta.yaml:48]: nope this is great - thank you!

**Pull request merged**
